docker-compose-staging.yml

```diff
@@ -130,7 +130,8 @@ services:
       - '--log.level=info'
       - '--web.telemetry-path=/statsd/metrics'
     ports:
-      - '9125:9125'
+      - '9125:9125/udp'
+      - '9125:9125/tcp'
```

Comment:
@burnerlee why is this needed?

Reply:
@Ram81 by default, docker only exposes the tcp port, while the statsd-exporter listens to both the tcp and udp port. So I have to explicity mention both of them for both of them to be exposed